lib/core/cardano-wallet-core.cabal

```diff
@@ -66,6 +66,7 @@ library
   exposed-modules:
       Cardano.Environment
       Cardano.Wallet
+      Cardano.Wallet.Compatibility
```

Comment:
This is temporary, once all the necessary refactored are completed, I'll create a new **package** `lib/http-bridge` which will contain all the Byron http-bridge-specific code, leaving `lib/core` completely agnostic to the network binary format and state being used.